app/controllers/dashboard_controller.rb

```diff
@@ -162,12 +158,10 @@ def watchlist
                .take(10)
   end
 
-
   def exports
     @bulk_exports = current_user.bulk_exports.order('id DESC').paginate :page => params[:page], :per_page => PAGES_PER_SCREEN
   end
 
-
   # Collaborator Dashboard - activity
   def editor
     @user = current_user
```

Comment:
I don't see this being used anywhere. Can you confirm?

Reply:
Do you mean the `editor` method?  (GH review is glitchy for me today)

Reply:
Yes, the `def editor` method